src/app.html

```diff
@@ -5,6 +5,7 @@
 		<link rel="icon" href="%sveltekit.assets%/favicon.ico" />
 		<meta name="viewport" content="width=device-width" />
 		<meta property="og:image" content="/imgs/og-img.png" />
+		<link rel="alternate" type="application/rss+xml" title="Hyprland News" href="/rss" />
```

Comment:
```suggestion
		<link rel="alternate" type="application/rss+xml" title="Hyprland News" href="/rss/" />
```

All the links that I've seen in the HEAD are to `rss/` and not `rss`.

So maybe this should work with the

> I tried to subscribe to the localhost address, so maybe they dislike that?

Reply:
EDIT: One example

https://blog.start9.com/

Reply:
`/rss` should be fine, as this will return `xml`.
In your example when I add `https://blog.start9.com/rss` to `NewsFlash` or `GFeeds` those apps still not show the articles.
So I might not really understand how Rss works :D

Reply:
Just try to add `blog.start9.com`. It works in NetNewsWire (not using this anymore) and Feeder (where I got the config in my Android through a `.opml` export and inspection)